src/lib/esnext.string.d.ts

```diff
@@ -0,0 +1,11 @@
+interface String {
+    /**
+     * Returns true if the string does not contain any lone UTF-16 surrogates.

```

Comment:
I'd rather use the description in the proposal:

```suggestion
     * Returns true if all leading surrogates and trailing surrogates appear paired and in order.
```